cli/cluster/cluster.go

```diff
@@ -239,10 +301,22 @@ func GetClusterConfig(path string) (ClusterConfig, error) {
 	collector := NewFileCollector(path)
 	config, err := collector.Collect()
 	if err != nil {
-		fmtErr := "unable to get cluster config from %q: %s"
+		fmtErr := "unable to get cluster config from %q: %w"
 		return ret, fmt.Errorf(fmtErr, path, err)
 	}
 
+	clusterConfig, err := MakeClusterConfig(config)
+	if err != nil {
+		return ret, fmt.Errorf("unable to parse cluster config from file: %w", err)
+	}
+	if len(clusterConfig.Config.Etcd.Endpoints) > 0 {
+		etcdConfig, err := collectEtcdConfig(clusterConfig)
```

Comment:
Environment may contain TT_CONFIG_ETCD* env vars, which, I suppose, must be merged to `Config.Etcd` before `collectEtcdConfig`
Please add an integration test for this.

Reply:
Added: `TestGetClusterConfig_etcd_connect_from_env`